frontend/app/utils/search.ts

```
@@ -76,35 +93,46 @@
       params.append(`${prefix}${this.keyMap.operator}`, filter.operator);
-      if (filter.value) {
-        filter.value.forEach((v, i) => params.append(`${prefix}${this.keyMap.value}[${i}]`, v || ''));
-      }
-      if (filter.filters) {
-        filter.filters.forEach((f, i) => addFilterParams(f, `${prefix}${this.keyMap.filters}[${i}].`));
-      }
+      filter.value?.forEach((v, i) =>
+        params.append(`${prefix}${this.keyMap.value}[${i}]`, v || '')
+      );
+      filter.filters?.forEach((f, i) =>
+        addFilterParams(f, `${prefix}${this.keyMap.filters}[${i}].`)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)